torch/distributed/elastic/utils/distributed.py

```diff
@@ -37,6 +37,12 @@ def create_c10d_store(
     retries=3,
     use_libuv: Optional[bool] = None,
 ):
+    if use_libuv is not None:
+        logger.warning("argument use_libuv is deprecated and ignored.")
```

Comment:
Hi, I think it's more user friendly if there's a hint what should be used (USE_LIBUV I think?) in warning message, and what 0 / 1 stands for, thanks!

Reply:
addressed in https://github.com/pytorch/pytorch/pull/135062